public/app/plugins/datasource/prometheus/configuration/PromSettings.tsx

```diff
@@ -116,6 +143,85 @@ export const PromSettings = (props: Props) => {
           )
         }
       />
+      {options.jsonData.flavour === THANOS && (
+        <>
+          <h3 className="page-heading">Thanos downsampling and retention</h3>
+          <div className="gf-form-group">
+            <div className="gf-form-inline">
+              <div className="gf-form max-width-30">
+                <FormField
+                  label="Retention for raw data"
+                  labelWidth={14}
+                  tooltip="Raw data retention. 0 for unlimited."
+                  inputEl={
+                    <Input
+                      className="width-25"
+                      value={retentionPolicies[DOWNSAMPLE_RAW.toString(10)] || '0d'}
+                      onChange={onRetentionChangeHandler(
+                        DOWNSAMPLE_RAW.toString(10),
+                        options,
+                        onOptionsChange,
+                        retentionPolicies
+                      )}
+                      validationEvents={promSettingsValidationEvents}
+                      spellCheck={false}
+                      placeholder="0"
+                    />
+                  }
+                />
+              </div>
+            </div>
+            <div className="gf-form-inline">
+              <div className="gf-form max-width-30">
+                <FormField
+                  label="Retention for 5m resolution"
```

Comment:
What does _m_ stand for here?

Reply:
Thank you. It did stand for minutes, as in PromQL duration. You're right though, it is unnecessary here. I've replaced promql abbreviations with English.